src/derivable/atom.ts

```diff
@@ -12,20 +12,23 @@ const logger = Logger.get('@politie/sherlock.atom');
  * with the initial state.
  */
 export class Atom<V> extends Derivable<V> {
+
+    /**
+     * @internal
+     * Contains the current value of this atom. Note that this field is public for transaction support, should
+     * not be used in application code. Use {@link Derivable#get} and {@link Atom#set} instead.
+     */
+    public value: V;
```

Comment:
We don't use `public` in the rest of the code. Please conform to the style. Also, the rest of the public properties are below the `constructor`, so move it to just below the constructor.